CHANGELOG.md

```diff
@@ -1,4 +1,27 @@
 ## 4.60.0 (Unreleased)
+
+FEATURES:
+* **New Data Source:** `google_compute_region_instance_template` ([#14132](https://github.com/hashicorp/terraform-provider-google/pull/14132))
+* **New Resource:** `apigeeKeystoresAliasesKeyCertFile` ([#14130](https://github.com/hashicorp/terraform-provider-google/pull/14130))
```

Comment:
The resource name is incorrectly formatted here

Reply:
Changed.